### ISSUE TITLE
Test bufr2ioda atmospheric dump files in fv3jedi_var.x

### ISSUE THREAD

**Issue by RussTreadon-NOAA** [NOAA-EMC/GDASApp#670]
This issue documents `fv3jedi_var.x` runs using dump files generated via g-w job `prepatmiodaobs` using bufr2ioda scripts.

This issue is a placeholder until the BUFR to IODA code sprint.  The code sprint will generate observation type specific issues.

**Comment by RussTreadon-NOAA** [comment]
Tests reported in this issue are for 2021081418 gdas.   GDASApp `develop` @ f9bddeb is used with g-w `develop` at [`2d64b74`](https://github.com/NOAA-EMC/global-workflow/commit/2d64b74a0b0407d01d6f657efa6af9637a131448).

GDASApp `develop` at f9bddeb supports atmospheric bufr to ioda processing for

`   
 BUFR_py = ["satwind_amv_goes", "satwind_scat", "adpupa_prepbufr"]
`

Job `gdasprepatmiodaobs` is run once for 2021081418 to generate bufr to ioda dump files for the observation types listed above.   The following is then done for each observation type

- `parm/atm/obs/lists/gdas_prototype_3d.yaml` is edited to only process a single bufr to ioda observation type
- rocotorewind and rocotoboot 2021081418 gdasatmanlinit
- rocotorewind and rocotoboot 2021081418 gdasatmanlrun

**Comment by RussTreadon-NOAA** [comment]
**satwind.abi_goes-16**
`fv3jedi_var.x` successfully runs to completion when processing `gdas.t18z.satwind.abi_goes-16.2021081418.nc4`

**scatwind_ascat_metop-a**
`fv3jedi_var.x` aborts with the message below when processing `gdas.t18z.scatwind.ascatw_metop-a.2021081418.nc4`
```
 10: Exception:         Reason: An exception occurred inside ioda while opening a variable.
 10:    name:   MetaData/stationElevation
 10:    source_column:  0
 10:    source_filename:        /work2/noaa/da/rtreadon/git/global-workflow/test2/sorc/gdas.cd/ioda/src/engines/ioda/src/ioda/Has_Variables.cpp
 10:    source_function:        ioda::Variable ioda::detail::Has_Variables_Base::open(const std::__cxx11::basic_string<char, std::char_traits<char>, std::allocator<char>> &) const
 10:    source_line:    108
 10:
```
Use `ncdump` to examine the contents of `gdas.t18z.scatwind.ascatw_metop-a.2021081418.nc4` (bufr-to-ioda) and `gdas.t18z.scatwind.2021081418.nc4` (gsidiags).   The gsidiags dump file contains `float stationElevation(Location)` in the `MetaData` group.   The bufr-to-ioda dump file does not contain this information.  How do we resolve this?  (1) Add `stationElevation` to the bufr-to-ioda file, (2) change the input yaml, (3) modify JEDI code, (4) a combination of 1, 2, and/or 3, or (5) do something else.

**adpupa_prepbufr**
`fv3jedi_var.x` hangs with the following snippets written to the job log

```
  0: GeometryData: Global tree size = 884736, footprint = 56623104
  0: sondes: read database from /work2/noaa/stmp/rtreadon/RUNDIRS/pratmos/gdasatmanl_18/obs/gdas.t18z.prepbufr.2021081418.nc4 (io pool size: 384)
  0: sondes processed vars: 8 variables: airTemperature, height, specificHumidity, stationPressure, verticalSignificance, virtualTemperature, windEastward, windNorthward
  0: sondes assimilated vars: 5 variables: stationPressure, airTemperature, windEastward, windNorthward, specificHumidity

... 

  0: 3DVar window: begin = 2021-08-14T15:00:00Z, end = 2021-08-14T21:00:00Z
  0: OOPS_STATS IncrementalAssimilation start            - Runtime:    14.04 sec,  Local Memory:   368.38 Mb
  0: Running incremental assimilation with 1 outer iterations.
  0: Minimizer algorithm=DRIPCG
  0: QNewtonLMP: maxouter : 1
  0: IncrementalAssimilation: Configuration for outer iteration 0:

...

  0:  cubic_grid domain decomposition
  0: whalo =    3, ehalo =    3, shalo =    3, nhalo =    3
  0:   X-AXIS =   48  48  48  48  48  48  48  48
  0:   Y-AXIS =   48  48  48  48  48  48  48  48
  0: GeometryData: Global tree size = 884736, footprint = 56623104
slurmstepd: error: *** JOB 15339905 ON Orion-18-07 CANCELLED AT 2023-10-10T14:23:34 DUE TO TIME LIMIT ***
  0: slurmstepd: error: *** STEP 15339905.0 ON Orion-18-07 CANCELLED AT 2023-10-10T14:23:34 DUE TO TIME LIMIT ***
```
It's possible program execution is further along but the buffer was not flushed to stdout.

**Comment by RussTreadon-NOAA** [comment]
Partially addressed by PR #937
